src/components/app.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import styles from './styles.css';
+
+class App extends Component {
```

Comment:
Yes, but I'm avoiding the issue of the livereload having this stateful component in the chain